### ISSUE TITLE
Diigolet can't be dropped into Favicon Bar?

### ISSUE THREAD

**Issue by Ryanrpm** [umfana/FaviconBar#16]
First off, thank you for creating this Favicon Bar for Safari!!

There is only one issue I've found with it though.  Apparently, the Diigolet link has several semicolons throughout it and when dropped into the Favicon bar it gets broken up into 3 or 4 different broken favicons.  :(  Please help.

If I could do away with my bookmarks bar and only keep the Favicon bar, I would!  But for now I have to keep both because of Diigolet.  See picture.
![favicon and bookmark bar](https://cloud.githubusercontent.com/assets/20267481/16545966/634fd578-40ef-11e6-939f-dc55b0194607.jpg)

**Comment by RoryCombe** [comment]
Glad you like the extension. Currently the bar isn't designed to be used with Javascript bookmarklets like the Diigolet. Unfortunately I don't have the time at the moment to implement the changes that would support Javascript links. 

I'm assuming you aren't a programmer that would be willing to contribute and code up the changes needed. If I'm wrong, I'd be more than happy to brief you on what needs to be done.

**Comment by Ryanrpm** [comment]
You are correct, I’m not a programmer.  I’ll just have to be patient until you have the time to make the changes to work with Javascript links  Thanks for all you have done…you’ve made using Safari more enjoyable and user friendly!

Ryan

**Comment by popcornhax** [comment]
I've got this on my todo list for a future release but I'm not 100% sure it can be done.  We'll see.
